src/test/suite/participant/participant.test.ts

```diff
@@ -1055,6 +1055,24 @@ suite('Participant Controller Test Suite', function () {
           });
         });
       });
+
+      suite('docs command', function () {
+        beforeEach(function () {
+          sendRequestStub.onCall(0).resolves({
+            text: ['connection info'],
+          });
+        });
+
+        test('falls back to the generic command when docs chatbot api is not available', async function () {
```

Comment:
We probably want a test that ensures we're doing the inverse as well, we use the docs chatbot when it is available.